Index: WoeUSB/configure.ac
===================================================================
--- WoeUSB/configure.ac
+++ WoeUSB/configure.ac	2017-12-22 11:50:51.416484872 +0530
@@ -1,4 +1,4 @@
-AC_INIT(woeusb, @@WOEUSB_VERSION@@)
+AC_INIT(woeusb, 3.1.4)
 AC_CONFIG_SRCDIR([src])
 AM_INIT_AUTOMAKE([foreign])
 AC_CONFIG_MACRO_DIR([m4])
Index: WoeUSB/src/woeusb
===================================================================
--- WoeUSB/src/woeusb
+++ WoeUSB/src/woeusb	2017-12-22 11:50:51.456484710 +0530
@@ -113,7 +113,7 @@
 	local -n only_for_gui_ref="$1"
 
 	declare -r application_name="WoeUSB"
-	declare -r application_version="@@WOEUSB_VERSION@@"
+	declare -r application_version="3.1.4"
 
 	declare -r DEFAULT_NEW_FS_LABEL="Windows USB"
 
Index: WoeUSB/src/woeusb.1
===================================================================
--- WoeUSB/src/woeusb.1
+++ WoeUSB/src/woeusb.1	2017-12-22 11:50:51.468484662 +0530
@@ -1,4 +1,4 @@
-.TH woeusb 1 "@@WOEUSB_VERSION@@" "WoeUSB" "WoeUSB User Manual"
+.TH woeusb 1 "3.1.4" "WoeUSB" "WoeUSB User Manual"
 .SH NAME
 woeusb \- Bootable Windows installation/PE USB storage creator
 
Index: WoeUSB/src/woeusbgui.1
===================================================================
--- WoeUSB/src/woeusbgui.1
+++ WoeUSB/src/woeusbgui.1	2017-12-22 11:50:51.481484609 +0530
@@ -1,4 +1,4 @@
-.TH woeusbgui 1 "@@WOEUSB_VERSION@@" "WoeUSB" "WoeUSB User Manual"
+.TH woeusbgui 1 "3.1.4" "WoeUSB" "WoeUSB User Manual"
 .SH NAME
 woeusbgui \- A GUI wrapper for woeusb(1)
 
Index: WoeUSB/debian/changelog
===================================================================
--- WoeUSB/debian/changelog
+++ WoeUSB/debian/changelog	2017-12-22 11:50:51.428484824 +0530
@@ -1,4 +1,4 @@
-woeusb (@@WOEUSB_VERSION@@) unstable; urgency=low
+woeusb (3.1.4) unstable; urgency=low
 
   * This is a fake changelog for packaging purpose, for the actual changelog pl-
     ease refer Commits · slacka/WoeUSB
